spec/features/maneuver_finding_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe 'finding a participant' do
+  let!(:maneuver) { create :maneuver }
+  let!(:record_1) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  let!(:record_2) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  let!(:record_3) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  context 'finds the previous' do
+    it 'called on first participant gets first participant' do
```

Comment:
"It called on first participant gets first participant" still isn't a sentence, unfortunately.